_rules/object-has-acessible-name-8fc3b6.md

```diff
@@ -38,12 +41,20 @@ The `object` element is not rendered for presentational purposes. If the `object
 
 ## Accessibility Support
 
-Non-supported media formats make screen readers render the text content of the element instead of other attributes.
+There are no major accessibility support issues known for this rule.
 
 ## Background
 
 Testing that the [accessible name][] describes the purpose of the `object` element is not part of this rule and must be tested separately.
 
+Non-supported media formats make screen readers render the text content of the element instead of other attributes. 
+
+`Object` elements without an accessible name are ignored by assistive technologies unless they have an [explicit role][].
+
+When the object resource is not loaded the fallback content is rendered as shown in Inapplicable Example 7.
+
+Valid `data` attributes define the `object` [accessible name][] that is announced by the screen reader.  The [accessible name][] is defined depending on the type of the `data` attribute. For example,  an image will use an `alt` for the [accessible name][] but a MIME will not.
```

Comment:
I'm still not fully understanding the details here. Especially what "MIME" means here. MIME types are a standard to declare type of data, and `image` is a standard MIME type.
So `<object data="foo.jpg" />` would have a MIME type of `image/jpeg`. Does this use `alt` for its accessible name?

Do you mean that "`image` MIME types uses `alt`, but other MIME types don't"?

Can you add test cases with this? It seems there is no test case with a `alt` attribute.